samples/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -233,6 +247,41 @@ static bool event_handler(const struct event_header *eh)
 		return false;
 	}
 
+	if (IS_ENABLED(CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE)) {
+		if (is_config_forward_event(eh)) {
+			struct config_forward_event *event = cast_config_forward_event(eh);
```

Comment:
this code seems rather long - maybe can be moved to separate function

Reply:
Reorganized, but I'm unwilling to move it to a separate function. Now we have 1st hid client call, then create event only if it is needed and 2nd hid client call.